mediabridge-frontend/src/components/MovieItem.tsx

```diff
@@ -23,6 +23,7 @@ const MovieItem = ({
             onClick={onRemove}
             className="w-6 h-6 rounded-full bg-red-500 text-white text-xs flex items-center justify-center hover:bg-red-600 transition-colors"
             aria-label="Remove movie"
+            name={`remove-${movie.title}`}
```

Comment:
I understand that you added this for the test, but its a bit brittle and awkward. What if the title has a quote in it or something, won't that break our HTML?

You can use the Cypress `.first()` method to easily find the first button on the page that has a class. So you could add `class="remove-button"` and then do `cy.get('.remove-button').first()` in your test (assuming that when you add the movie there is only one remove button on the screen).